src/radical/pilot/utils/db_utils.py

```diff
@@ -55,46 +52,19 @@ def get_last_session(db):
 
 
 # ------------------------------------------------------------------------------
-def get_session_docs(db, sid, cache=None, cachedir=None):
+def get_session_docs(sid, cachedir=None):
```

Comment:
this what causes the issue when is called here
https://github.com/radical-cybertools/radical.pilot/blob/d20b4f614f5470abc8b142dbfd51a26934ce2528/src/radical/pilot/utils/session.py#L84-L86

Reply:
Thanks, @mtitov for checking but why the error does not show up on the local test?

Reply:
Only can guess that different branch (or the current one, but with local changes) is used or a different example was running. Seems like this issue couldn't be just skipped, you suppose to have it locally as well

Reply:
Just found it, the issue is deeper than I expected, the session is creating a `json` file upon the closing time that must be fetched/created from `mongodb` as a MUST. `session.py` uses `get_session_doc` to do that and we removed this functionality (did see that) unfortunately, I must fix a few things.